packages/stokhos/CMakeLists.txt

```diff
@@ -28,7 +28,7 @@ TRIBITS_ADD_OPTION_AND_DEFINE(${PACKAGE_NAME}_ENABLE_Experimental
 TRIBITS_ADD_OPTION_AND_DEFINE(${PACKAGE_NAME}_ENABLE_Ensemble_Reduction
   HAVE_STOKHOS_ENSEMBLE_REDUCT
   "Enable reduction across across ensemble components."
-  ON )
+  OFF )
```

Comment:
I don't see how this is relevant to this PR.

Reply:
It's relevant because this whole PR is about making MueLu work when magnitudeType != double, and this change makes that the default case for Stokhos, to ensure it doesn't break in the future.

Reply:
@etphipp thanks for explaining!